plugin.video.wnbc/resources/settings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+
+	<category label="30012">
+		<setting type="sep" />
+		<setting id='enable_meta' type ="bool" label="30013" default="true"/>
+           <setting id="init_meta" type="bool" label="30014" default="true" enable="!eq(-1,false)"/>
+		<setting type="sep" />
+	</category>
+
+  <category label="30020">
```

Comment:
I can remove it, but there will be many user complaints. Shall I direct them to you?